blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -312,6 +365,19 @@ Nav.propTypes = {
       label: 'Horizontal Links hierarchy',
       group: 'Configure content',
     }),
+    desktopNavivationStartHeight: PropTypes.number.tag({
+      label: 'Starting desktop navigation bar height',
+      group: 'Logo',
+      max: 200,
+      min: 56,
```

Comment:
Following this documentation, min and max should works. But I could not test that on my local, there're no constraint when user input the value in page builder editor.

https://appledaily.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/api/feature-pack/components/custom-fields.md

Reply:
Based on the conversation in standup, it sounds like this is a bug in Fusion. You're saying that the user can put in any number even if it's smaller than the `min` or larger than the `max` right?

Reply:
Yes, I can put any value in that custom field